### ISSUE TITLE
Signup: inconsistent padding for Signup flow on mid-size viewports

### ISSUE THREAD

**Issue by StefanNieuwenhuis** [Automattic/wp-calypso#54972]
### Why
Inconsistencies in the UI/UX of any flow, but the signup flow in particular, causes confusion among users. We want to prevent this and provide a _happy user journey_ for all.

### What
There are some breakpoints where padding is inconsistent in the reskin variant of the signup flow.
Video: https://video.wordpress.com/v/sDOAzN2y

### Acceptance criteria
-[ ] TBD

### Notes
- This issue is addressed in p1627047728264300-slack-C01VA100LEA & p1626801132247300-slack-C01VA100LEA
- Padding inconsistencies for the Domain Step are addressed in https://github.com/Automattic/wp-calypso/issues/54131

**Comment by daledupreez** [comment]
I am jumping in to note that the new Email step my team has been working on started from the Domain step CSS -- I'll follow that PR, as I think we should use the same approach for both steps.

**Comment by StefanNieuwenhuis** [comment]
> I am jumping in to note that the new Email step my team has been working on started from the Domain step CSS -- I'll follow that PR, as I think we should use the same approach for both steps.

I agree. The best thing is to use the same CSS classes for both steps wherever we can.

**Comment by jordesign** [comment]
Hey @daledupreez  - quick gut check. Do you think this issue is still relevant, or have we moved sufficiently away from this onboarding flow that it is no longer needed?
